indico/modules/events/surveys/blueprint.py

```diff
@@ -19,16 +19,25 @@
 from indico.modules.events.surveys.controllers.management.results import (RHDeleteSubmissions, RHDisplaySubmission,
                                                                           RHExportSubmissionsCSV,
                                                                           RHExportSubmissionsExcel, RHSurveyResults)
-from indico.modules.events.surveys.controllers.management.survey import (RHCloseSurvey, RHCreateSurvey, RHDeleteSurvey,
-                                                                         RHEditSurvey, RHManageSurvey, RHManageSurveys,
-                                                                         RHOpenSurvey, RHScheduleSurvey,
-                                                                         RHSendSurveyLinks)
+from indico.modules.events.surveys.controllers.management.survey import (RHAPIEmailEventSurveyMetadata,
+                                                                         RHAPIEmailEventSurveySend, RHCloseSurvey,
+                                                                         RHCreateSurvey, RHDeleteSurvey, RHEditSurvey,
+                                                                         RHEmailEventSurveyPreview, RHManageSurvey,
+                                                                         RHManageSurveys, RHOpenSurvey,
+                                                                         RHScheduleSurvey, RHSendSurveyLinks)
 from indico.web.flask.wrappers import IndicoBlueprint
 
 
 _bp = IndicoBlueprint('surveys', __name__, template_folder='templates', virtual_template_folder='events/surveys',
                       url_prefix='/event/<int:event_id>', event_feature='surveys')
 
+_bp.add_url_rule('api/surveys/<int:survey_id>/email/metadata', 'api_email_event_survey_metadata',
+                 RHAPIEmailEventSurveyMetadata, methods=('POST',))
+_bp.add_url_rule('api/surveys/<int:survey_id>/email/preview', 'api_email_event_survey_preview',
+                 RHEmailEventSurveyPreview, methods=('POST',))
+_bp.add_url_rule('api/surveys/<int:survey_id>/email/send', 'api_email_event_survey_send',
+                 RHAPIEmailEventSurveySend, methods=('POST',))
```

Comment:
```suggestion
_bp.add_url_rule('/api/surveys/<int:survey_id>/email/metadata', 'api_email_event_survey_metadata',
                 RHAPIEmailEventSurveyMetadata, methods=('POST',))
_bp.add_url_rule('/api/surveys/<int:survey_id>/email/preview', 'api_email_event_survey_preview',
                 RHEmailEventSurveyPreview, methods=('POST',))
_bp.add_url_rule('/api/surveys/<int:survey_id>/email/send', 'api_email_event_survey_send',
                 RHAPIEmailEventSurveySend, methods=('POST',))
```

Otherwise you get SUPER weird URLs, like this:

```
/event/42api/surveys/69/email/metadata
```

But of course you want this:

```
/event/42/api/surveys/69/email/metadata
```